libraries/ms-helper/inc/exported_enums.h

```diff
@@ -30,3 +30,10 @@ typedef enum {
   EE_RELAY_STATE_CLOSE,
   NUM_EE_RELAY_STATES,
 } EERelayState;
+
+typedef enum {
```

Comment:
Can you add a super brief comment with what this data field is expected to be used with?

(i.e. driver controls tx?)

Reply:
Done.